cmd/hashers/argon2/loadtest.go

```diff
@@ -0,0 +1,257 @@
+package argon2
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/ory/x/flagx"
+
+	"github.com/fatih/color"
+	"github.com/inhies/go-bytesize"
+	"github.com/montanaflynn/stats"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/ory/kratos/hash"
+	"github.com/ory/x/cmdx"
+	"github.com/ory/x/configx"
+)
+
+type resultTable struct {
+	Total  time.Duration     `json:"total"`
+	Median time.Duration     `json:"median"`
+	StdDev time.Duration     `json:"std_deviation"`
+	Min    time.Duration     `json:"min"`
+	Max    time.Duration     `json:"max"`
+	MaxMem bytesize.ByteSize `json:"max_mem"`
+}
+
+var (
+	ErrSampleTimeExceeded        = fmt.Errorf("the sample time was exceeded")
+	ErrMemoryConsumptionExceeded = fmt.Errorf("the memory consumption was exceeded")
+
+	_ cmdx.TableRow = &resultTable{}
+)
+
+func (r *resultTable) Header() []string {
+	return []string{"TOTAL SAMPLE TIME", "MEDIAN", "STANDARD DEVIATION", "MIN", "MAX", "MEMORY USED"}
+}
+
+func (r *resultTable) Columns() []string {
+	return []string{
+		r.Total.String(),
+		r.Median.String(),
+		r.StdDev.String(),
+		r.Min.String(),
+		r.Max.String(),
+		r.MaxMem.String(),
+	}
+}
+
+func (r *resultTable) Interface() interface{} {
+	return r
+}
+
+func newLoadTestCmd() *cobra.Command {
+	flagConf := &argon2Config{}
+
+	cmd := &cobra.Command{
+		Use:  "load-test <authentication-requests-per-minute>",
+		Args: cobra.ExactArgs(1),
```

Comment:
This is missing a description. The output is also not very clear. What parameters are being used here?

```
 $GOPATH/bin/kratos hashers argon2 load-test 30 
It takes about 20s to collect all necessary data, please be patient.
TOTAL SAMPLE TIME	16.406087875s	
MEDIAN			207.022782ms	
STANDARD DEVIATION	64.05794ms	
MIN			145.107023ms	
MAX			344.1785ms	
MEMORY USED		1.51GB	
```

Reply:
The ones defined in the config file/defaults/flags/... Just as kratos would use them if you would run `kratos serve`
Should they be printed as well, just to indicate and help to spot mistakes?